src/zc/table/column.txt

```diff
@@ -224,10 +224,11 @@ The input function also validates input.  If there are any errors, a
 WidgetsError will be raised:
 
     >>> request.form["test.NA==.email"] = u'walsh'
-    >>> data = columns[1].input(contacts, request)
-    Traceback (most recent call last):
-    ...
-    WidgetsError: WidgetInputError:
+    >>> try:
+    ...     data = columns[1].input(contacts, formatter)
+    ... except zope.formlib.interfaces.WidgetsError as e:
+    ...     e
+    WidgetInputError:
         ('email', u'', ConstraintNotSatisfied(u'walsh', 'email'))
```

Comment:
I don't quite understand this change.  Are there differences in the exception formatting that doctest is unable to cope with?

Reply:
There is doctest.IGNORE_EXCEPTION_DETAIL which does exactly that. I'll revert this change and add the flag.